db/dbformat.h

```diff
@@ -81,8 +81,8 @@ struct ParsedInternalKey {
   ValueType type;
 
   ParsedInternalKey()
-      : sequence(kMaxSequenceNumber)  // Make code analyzer happy
-  {}  // Intentionally left uninitialized (for speed)
+      : sequence(kMaxSequenceNumber),  // Make code analyzer happy
+        type(kTypeDeletion) { } // Intentionally left uninitialized (for speed)
```

Comment:
Can you revert the change for dbformat.h? As the comment said it is left uninitialized for speed, although I don't know how much difference there is.

Reply:
@yiwu-arbug made the changes as requested. Kindly review the changes.